components/AddToCart.js

```diff
@@ -0,0 +1,71 @@
+import cookie from 'js-cookie';
+import React from 'react';
+
+export default function AddToCart(props) {
+  const itemsInCart = cookie.getJSON('cart') || [];
+  console.log('jonas', itemsInCart);
+
+  // useEffect(() => {
+  //   cookie.set('cart', itemsInCart);
+  //   console.log(itemsInCart);
+  // }, [itemsInCart]);
+
+  function MakeCookies() {
+    const product = {
+      name: props.product.name,
+      price: props.total,
+      id: props.product.id,
```

Comment:
Not really a problem, but very often you will see `id` as the first key of the object, so that you can see right away that it has an `id`.